### PR TITLE
feat(collections-ddl): add time-series support COMPASS-4778

### PR DESCRIPTION
**Pull request by mcasimir**
- Allow to create timeseries collections from the create collection dialog if server >= v5.0
- Add leafygreen to the create collection dialog
- Add properties badges for collation, view, timeseries, read-only and capped to the collections table
- Fix https://jira.mongodb.org/browse/COMPASS-3921

### `< 5`

https://user-images.githubusercontent.com/334881/118874741-87cb3900-b8eb-11eb-9aa4-cdbc08f8075e.mp4

### `>= 5`

https://user-images.githubusercontent.com/334881/118874752-8994fc80-b8eb-11eb-98f9-c34cc49cb056.mp4

### Create modal

https://user-images.githubusercontent.com/334881/118876280-5c494e00-b8ed-11eb-9092-6b9403ece510.mp4

## Notes / Known issues (already present in master):
- `capped` is always `false` when a collection is loaded
- creating and deleting a collection brings you back to the wrong screen (database list)
- creating a collection with collation fails in development with a seemingly unrelated error that suggests that the wrong command is invoked, could be a webpack issue.

Screenshot 2021-05-19 at 21 37 56

### CODE REVIEW

File: packages/compass-collections-ddl/src/components/create-collection-modal/create-collection-modal.jsx [hunk 1]
```diff
@@ -119,12 +135,26 @@ class CreateCollectionModal extends PureComponent {
...
+    }
+  }
+
+
+  renderTimeSeries() {
+    if (this.props.isTimeSeries) {
+      return (
+        <div className={styles['create-collection-modal-is-time-series-wrapper']}>
+          <TimeSeries changeTimeSeriesOption={this.props.changeTimeSeriesOption}
+            timeSeries={this.props.collation}/>
```
Comment (Anemy) [inline packages/compass-collections-ddl/src/components/create-collection-modal/create-collection-modal.jsx:157]: Should this be `timeSeries={this.props.timeseries}`?

Comment (mcasimir) [reply packages/compass-collections-ddl/src/components/create-collection-modal/create-collection-modal.jsx:157]: Yeah .. nice catch, wondering how this causes no issue 😕

Comment (mcasimir) [reply packages/compass-collections-ddl/src/components/create-collection-modal/create-collection-modal.jsx:157]: wait a minute .. that prop is useless i think, unless we would ever do an "edit" dialog, cause leafygreen fields don't need the prop value to change content while you type. i can probably just ditch it entirely.

Comment (mcasimir) [reply packages/compass-collections-ddl/src/components/create-collection-modal/create-collection-modal.jsx]: Ok i've removed it, let me know if you see issues with that.

**Review by Anemy — APPROVED** [review]
lgtm - nice job starting to pull some leafygreen into compass! Nice improvements to the dev env of this package.
Tried out making time series collections and works nice and smooth. Was also able to get that collation error on dev but not when packaging it pulling it into Compass. :/ 

I love seeing less of the unnecessary `classnames` and fewer webpack aliases :)

**Pull request merged**